server/src/grpc/macros.rs

```diff
@@ -1,4 +1,4 @@
 //! log macro's for gRPC logging
 
 use lib_common::log_macros;
-log_macros!("grpc");
+log_macros!("grpc", "app::grpc");
```

Comment:
Not sure why the prefix is specifically provided here. The macro defaults to `app` as prefix if nothing is provided.
Providing `app::grpc` as the prefix as is done here results into log messages in the namespace `app::grpc::grpc` (the first param will be appended to the namespace as well)

Reply:
https://github.com/aetheric-oss/svc-scheduler/pull/53/commits/8828aa5409cc759b31cd775b82e39728e8e4aba7